CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Deprecated `SyncResolveError` and `SyncResolveErrorCode` ([#1182](https://github.com/realm/realm-dart/pull/1182)).
 * Added `SyncWebSocketError` and `SyncWebSocketErrorCode` for web socket connection sync errors ([#1182](https://github.com/realm/realm-dart/pull/1182)).
 * Added `FlexibleSyncConfiguration.shouldCompactCallback` support ([#1204](https://github.com/realm/realm-dart/pull/1204)).
+* Added `RealmSet.asResults()` to match `RealmList` ([#1214](https://github.com/realm/realm-dart/pull/1214)).
```

Comment:
Why to match `RealmList`? Doesn't it return a `RealmResult`?

Reply:
Because `RealmList` also has a `asResults()` method?

Reply:
I have removed the _to match .._ part